.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,6 @@
+- id: jupytext
+  name: jupytext
+  description: Runs jupytext on all notebooks and paired files.
+  language: python
+  entry: jupytext --ignore-unmatched --alert-untracked
```

Comment:
Should this really be in entry or passed args. The user should be able to override this if they really want without modifying the entry, right? Thoughts @mwouts ?

Reply:
Note to self: Add proper types for file matching.

Reply:
No, these are both essential to the hook functioning imo. Other rags can be passed but it has to consistently fail if changes are made, and it should ignore input files that it's not trying to do anything with. Users can specify types/files if they want.